README.md

```diff
@@ -156,24 +215,106 @@ final CMAArray<CMAEntry> array =
         .fetchAll();
 ```
 
-#### Words of warning
+*Words of warning*
 
-Currently `apiKeys`, `environments`, `roles`, `spaceMemberships`, `uiExtensions`, `uploads`, and
-`webhooks`, do not support environments different to `master` (or Constants.DEFAULT_ENVIRONMENT). If
-you want to use the configuration explained above with theses endpoints, be aware that they will
-throw an exception symbolising that you cannot use them with a defined environment id. Please create
-a new client without specifying an environment id to perform an action like this.
+The Modules `apiKeys`, `environments`, `roles`, `spaceMemberships`, `uiExtensions`, `uploads`, and `webhooks`, do not support Environments different to `master`. If the configuration explained is used with these Modules, they will throw an `exception`. Creation of a new client without specifying an Environment id is needed and repeating of the ids in the call is needed:
+
+```
+final CMAArray<CMAApiKey> array = 
+    client
+        .apiKeys()
+        .fetchAll(
+	   "spaceid"
+	);
+```
+
+Default HTTP Client
+-------------------
+
+The SDK uses [Retrofit][2] as a REST client, which detects [OkHttp][3] in your classpath and uses it if it's available, otherwise falls back to the default `HttpURLConnection`.
```

Comment:
|Before|After|
|-----|-----|
| in your classpath and uses it if it's available, otherwise falls back to the default `HttpURLConnection`.| in the classpath and uses it if available, or else it falls back to the default `HttpURLConnection`.